### PR TITLE
[IMP] account_global_discount: Discounts in invoice report

### PR DESCRIPTION
**Pull request by chienandalu**
Give discounts info in the invoice report.
image

@Tecnativa TT20971

### CODE REVIEW

File: account_global_discount/views/report_account_invoice.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <xpath expr="//div[@id='total']/div/table/tr[1]" position="before">
+            <t t-if="o.global_discount_ids">
+                <tr>
+                    <td><strong>Subtl. before disc.</strong></td>
```
Comment (pedrobaeza) [inline account_global_discount/views/report_account_invoice.xml]: Weird abbreviation

```suggestion
                    <td><strong>Subtotal w/o disc.</strong></td>
```

Comment (chienandalu) [reply account_global_discount/views/report_account_invoice.xml]: Ok. Done

**Comment by chienandalu** [comment]
Changes done and added spanish translation

**Review by pedrobaeza — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
Thanks! :+1:

**Comment by pedrobaeza** [comment]
Let's merge it as straight-forward

/ocabot merge minor

**Pull request merged**